### PR TITLE
feat: add dummyio app

### PR DESCRIPTION
**Pull request by gi0baro**
SSIA

### CODE REVIEW

File: helper-images/dummyio/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+granian
```
Comment (tonyo) [inline helper-images/dummyio/requirements.txt:1]: Let's pin the version.

File: helper-images/dummyio/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM python:3.11-slim as builder
+
+RUN python -m venv .venv
+COPY requirements.txt .
+RUN .venv/bin/pip install -r requirements.txt
+
+FROM python:3.11-slim
+
+COPY --from=builder /.venv /.venv
```
Comment (tonyo) [inline helper-images/dummyio/Dockerfile:9]: Why do we even need staged builds if we copy the entire venv?

**Review by tonyo — APPROVED** [review]
Let's add a README of what this is and why we need it.

File: helper-images/dummyio/Dockerfile (hunk 2 above)
Comment (gi0baro) [reply helper-images/dummyio/Dockerfile]: Probably more a habit than anything else.
The only valuable point with such an easy image is that we don't keep requirements.txt in the final image.
Which might also be achieved with a `RUN rm`.
Does this bother us?

Comment (tonyo) [reply helper-images/dummyio/Dockerfile]: >  Which might also be achieved with a RUN rm.

`RUN rm` will only make the image bigger, though

Not a blocker, ok to leave.

**Pull request merged**